lib/features/about/contributor.model.dart

```diff
@@ -0,0 +1,7 @@
+class Contributor {
+  String name;
+  String email;
+  String linkedIn;
+
+  Contributor(this.name, this.email, this.linkedIn);
```

Comment:
Nice start !

Try to use named params to promote maintainability and self explain/explicit constructors
```suggestion
  Contributor({this.name, this.email, this.linkedIn});
```